iterators.tex

```diff
@@ -339,8 +344,8 @@
   template <class Out, class T>
   concept bool Writable() {
     return Semiregular<Out>() &&
-      requires(Out o, T&& v) {
-        *o = std::forward<T>(v); // not required to be equality preserving
+      requires(Out o, T&& t) {
+        *o = std::forward<T>(t); // not required to be equality preserving
```

Comment:
These changes in `Writable` are causing the conflict: I already replaced `v` with `t` in the concept definition, and I replaced `v` in the text with `E`, "an expression such that `decltype(E)` is `T`".